data/SMURFCAT/data.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Real Smurf Cat",
+  "symbol": "SMURFCAT",
+  "decimals": 18,
+  "description": "Smurfcat is a memecoin launched on the Ethereum blockchain. It's a tribute to the Smurfcat originally created by Nate Hallinan in 2011, and popularized by TikTok. It began as a fun currency and has evolved into a whole ecosystem revolving around the little guy. As the meme matures, the full Smurfcat story will be revealed. Join us and become part of a movement. Bringing together a community where fun, creativity and mycelium coexist.",
+  "website": "https://smurfcat.eth.limo/",
+  "twitter": "@smurfcateth",
+  "tokens": {
+    "ethereum": {
+      "address": "0xfF836A5821E69066c87E268bC51b849FaB94240C"
```

Comment:
getting closer :) You now need a "," after this line for it to be valid JSON.

Reply:
I really am blind, fixed 😹